### PR TITLE
Fixed issue #16230: Reload survey (token answer persistence) don't really reload 

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: LEMtokenResume with reload by token too
Dev: set maxstep to totalstep if needed
Dev: Create LimeExpressionManager::SetRelevanceTo to set relevance before JumpTo in case of reload
Dev: When reload : initFirstStep function before all other: then move reloading part here.

### CODE REVIEW

**Comment by Shnoulle** [comment]
@glimz In project management it set as done : https://github.com/LimeSurvey/LimeSurvey/pull/2211

**Review by gabrieljenik — CHANGES_REQUESTED** [review]
Some changes were requested throught the code.

File: application/helpers/expressions/em_manager_helper.php [hunk 1]
```diff
@@ -8767,9 +8788,20 @@ private function _GetVarAttribute($name, $attr, $default, $gseq, $qseq)
...
-                $sqrel = (isset($_SESSION[$this->sessid]['relevanceStatus'][$rowdivid]) ? $_SESSION[$this->sessid]['relevanceStatus'][$rowdivid] : 1);    // true by default - only want false if a subquestion is irrelevant
+                $grel =  1;
+                if(isset($_SESSION[$this->sessid]['relevanceStatus']['G' . $gseq])) {
+                    $grel =  $_SESSION[$this->sessid]['relevanceStatus']['G' . $gseq];
+                }
+                $qrel =  null;
+                if(isset($_SESSION[$this->sessid]['relevanceStatus'][$qid])) {
+                    $qrel =  $_SESSION[$this->sessid]['relevanceStatus'][$qid];
+                } else {
+                    // TODOACTUEL
```
[the remark this answers is not in the document]
Comment (Shnoulle) [reply application/helpers/expressions/em_manager_helper.php:8799]: Set it before :)

In fact ;: i want to create a new settings (global or not unsure) for "DefaultQuestionRelevanceStatus)

but it's another question here.

**Review by gabrieljenik — APPROVED** [review]
Code OK.
Not tested.

**Comment by Shnoulle** [comment]
Maybe need test ?

**Pull request merged**